cli/src/commands/root.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::io::Write;
+
+use tracing::instrument;
+
+use crate::cli_util::{user_error, CommandError, CommandHelper};
+use crate::ui::Ui;
+
+#[derive(clap::Args, Clone, Debug)]
+pub(crate) struct RootArgs {}
```

Comment:
Nit: Missing documentation.

Reply:
Done, I didn't realize this was user-facing.